mirage-os-shim.opam

```
@@ -4,32 +4,17 @@ authors: ["David Kaloper Meršinjak <david@numm.org>"]
 homepage: "https://github.com/mirage/mirage-os-shim"
 doc: "https://mirage.github.io/mirage-os-shim/doc"
 license: "ISC"
-dev-repo: "git+https://github.com/mirage/mirage-os-shim.git"
+dev-repo: "git+https://github.com/dune-universe/mirage-os-shim.git"
```

```suggestion
dev-repo: "git+https://github.com/mirage/mirage-os-shim.git"
```

this looks unintentional